site/app/models/OfficeHoursQueueModel.php

```diff
@@ -236,6 +236,31 @@ public function getCurrentTimePausedStart(): ?string {
         return $this->current_queue_state['time_paused_start'];
     }
 
+    public function getHelperName() {
+        return $this->getDisplayName($this->current_queue_state['helper_firstname'], $this->current_queue_state['helper_lastname'], $this->current_queue_state['helper_group']);
+    }
+
+    public function getDisplayName($first_name, $last_name, $group) {
+        $user_info = [];
+        $user_info["user_firstname"] = $first_name;
+        $user_info["user_lastname"] = $last_name;
+
+        // set to empty string because it is not used but is required by the User model
+        $user_info["user_id"] = "";
+        $user_info["user_email"] = "";
+        $user_info["user_email_secondary"] = "";
+        $user_info["user_email_secondary_notify"] = "";
+
+        $user = new User($this->core, $user_info);
+
+        if ($group <= 2) {
```

Comment:
What does the number 2 mean here?  It would be better to replace it with a constant which indicates its meaning.